odinfmt.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Oops, will remove. ols really wanted to format the file differently than your preference.